src/v2/features/characters/characters-value.service.ts

```diff
@@ -0,0 +1,474 @@
+import * as fs from 'fs';
+
+import { ICharacter2 } from 'src/models/interfaces';
+import { Rank, Rarity, RarityStars } from 'src/models/enums';
+
+const readDataFromFile = (filename: string): any => {
+    const rawData = fs.readFileSync(filename);
+    return JSON.parse(rawData.toString());
+};
+
+const rankUpData = readDataFromFile('assets/rankUpData.json');
+const recipeData = readDataFromFile('assets/recipeData.json');
```

Comment:
Import JSON data directly like it is done in the `characters-poser.service` with dirty dozen data
e.g.
`import rankUpData from 'src/assets/rankUpData.json';`